packages/devextreme-angular/tsconfig.json

```
@@ -1,9 +1,11 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "target": "es2015",
+    "target": "ES2020",
```

If there no problems with it, use 'ES2017'. We still can't use some features from 'ES2020' due to mobile browsers support
```suggestion
    "target": "ES2017",
```